controllers/provider-gcp/docs/usage-as-end-user.md

```diff
@@ -43,6 +45,9 @@ The `networks.vpc` section describes whether you want to create the shoot cluste
 * If `networks.vpc.name` is given then you have to specify the VPC name of the existing VPC that was created by other means (manually, other tooling, ...).
 If you want to get a fresh VPC for the shoot then just omit the `networks.vpc` field.
 
+* If a VPC is to be re-used, and the `networkers.vpc.cloudRouter.Name` is given then you have to specify the name of a CloudRouter that was created by other means (manually, other tooling, ...), 
```

Comment:
```suggestion
* If a VPC is to be re-used, and the `networkers.vpc.cloudRouter.name` is given then you have to specify the name of a CloudRouter that was created by other means (manually, other tooling, ...), 
```

Reply:
I would not understand this as an end-user, I think.
Can we rephrase such that the following gets clear:

* If a VPC name is not given then we will create the cloud router + NAT gateway to ensure that worker nodes don't get external IPs.
* If a VPC name is given then
  * if a cloud router name is not given we won't do anything, i.e. not creating a cloud router + NAT ourselves, resulting in worker nodes having external IPs (only for limited amount of time, end-users are asked to migrate to the following case)
  * if a cloud router name is given we re-use this one and create a dedicated NAT for the shoot, allowing worker nodes to not have external IPs.